### PR TITLE
Marius/table changes

### PR DESCRIPTION
**Pull request by EnvyPhD**
Major changes to display and style. Implemented changes together with Mali. Worked with Rebekka on implementing styles.

### CODE REVIEW

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx [hunk 1]
```diff
@@ -71,143 +59,115 @@ const PatientView = (props: {
...
-                                                }
-                                            </p>
+                                            calcAge()
                                         ) : (
                                             <div className="unavailable-content">
                                                 Ikke oppgitt
                                             </div>
                                         )}
-                                    </div>
+                                    </p>
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:103]: I think wrapping the div in a p gives warnings

Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:103]: Fixed

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss [hunk 2]
```diff
@@ -10,57 +10,57 @@
...
         }
         .info-left {
+            padding-right: 100px;
             p {
-                font-weight: bold;
-                margin-left: 1.6rem;
-                font-size: 1.25rem;
+                font-size: 1rem;
+                font-weight: 500;
             }
```
Comment (erlinvan) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss:37]: I there a reason we are not consistently using either rem or px for font size?

File: src/dataexplorer/dataexplorer-application/src/components/Navigation/Breadcrumbs/Breadcrumbs.tsx [hunk 3]
```diff
@@ -14,26 +9,23 @@ const Breadcrumbs = () => {
...
                                 <span className="breadcrumbs-item">{name}</span>
                             </Link>
                         </Breadcrumb.Item>
                     )}
                     {schemaNumber !== '' && (
                         <Breadcrumb.Item>
-                            <FileTextOutlined className="bread-icon" />
                             <span className="breadcrumbs-item">
-                                Skjema - {schemaNumber}
+                                <b>Skjema {schemaNumber}</b>
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Navigation/Breadcrumbs/Breadcrumbs.tsx:28]: The <b> tag is not recommended for styling. Suggest to use <strong> or font-weight instead

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss (hunk 2 above)
Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss:37]: We are using rem consequently. Forgot to change these fields before pushing this. Will change

File: src/dataexplorer/dataexplorer-application/src/components/Navigation/Breadcrumbs/Breadcrumbs.tsx (hunk 3 above)
Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Navigation/Breadcrumbs/Breadcrumbs.tsx:28]: 100% agree. This wasn't added now, so unsure why GIT says it was 🍶

**Pull request merged**